ansible/roles/aws_display_info/tasks/minimal.yml

```diff
@@ -2,17 +2,22 @@
 - debug: var=my_ec2_facts
 
 - block:
+    - debug:
```

Comment:
Nice, this greatly improves the EC2 instance information!

And could we also show if the proxy option is enabled, and display any information that may of use to the end user?